haiku-apps/conway/conway-0.9.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A simple implementation of \"Conway's Game of Life\""
```

Comment:
I think it would look nicer with the quotes before Game:
```diff
-SUMMARY="A simple implementation of \"Conway's Game of Life\""
+SUMMARY="A simple implementation of Conway's \"Game of Life\""
```
What do you think?